src/components/MostViewed.js

```diff
@@ -72,7 +62,29 @@ export default class MostViewed extends Component {
         return (
             <div>
                 <Heading>Most Viewed</Heading>
-                <Main dangerouslySetInnerHTML={{ __html: this.state.html }} />
+                <div>
```

Comment:
Do we need this wrapper `div`? Or the one beginning on line 69?

Reply:
Nope! Thanks.